File: Linear/Dynamic/Stack/Stack.java
import java.util.Stack;

class MyStack {
    private Stack<Integer> st = new Stack<>();

    public void push(int value) {
        st.push(value);
    }

    public void pop() {
        if (!st.isEmpty()) {
            st.pop();
        } else {
            System.out.println("Stack is empty");
        }
    }

    public int top() {
        if (!st.isEmpty()) {
            return st.peek();
        } else {
            System.out.println("Stack is empty");
        }
        return -1;
    }

    public boolean isEmpty() {
        return st.isEmpty();
    }

    public int size() {
        return st.size();
    }
}

class StackOperation {
    public static void main(String[] args) {
        MyStack st = new MyStack();

        st.push(1);
        st.push(2);
        st.push(3);

        System.out.println("Top: " + st.top());

        st.pop();
        System.out.println("After pop, top: " + st.top());

        System.out.println("Size: " + st.size());

        System.out.println("Is empty: " + (st.isEmpty() ? "Yes" : "No"));
    }
}

File: Linear/Dynamic/Stack/Stack.js
class Stack {
    constructor() {
        this.st = []
    }

    push(value) {
        this.st.push(value)
    }

    pop() {
        if (this.st.length > 0) {
            this.st.pop()
        } else {
            console.log("Stack is empty")
        }
    }

    top() {
        if (this.st.length > 0) {
            return this.st[this.st.length - 1]
        } else {
            console.log("Stack is empty")
        }
        return null
    }

    isEmpty() {
        return this.st.length === 0
    }

    size() {
        return this.st.length
    }
}

const st = new Stack()

st.push(1)
st.push(2)
st.push(3)

console.log("Top:", st.top())

st.pop()
console.log("After pop, top:", st.top())

console.log("Size:", st.size())

console.log("Is empty:", st.isEmpty() ? "Yes" : "No")